tests/cockpit-tests

```diff
@@ -31,17 +31,12 @@ if [ ! -d cockpit ]; then
     git clone https://github.com/cockpit-project/cockpit
 fi
 
+# TODO: This needs to move into tests bot
 if [ ! -f $TMPDIR/vm-prep-was-run ]; then
     sudo /usr/local/bin/vm-prep
     touch $TMPDIR/vm-prep-was-run
 fi
 
-if [ ! -d cockpit/node_modules ]; then
```

Comment:
A long time ago, we wouldn't load node_modules during autogen.sh. We started looping at the end of this script. That means the first autogen.sh in the container installs the node_modules anyway, without efficiency problems.